src/sidebar/test/cross-origin-rpc-test.js

```diff
@@ -12,7 +12,7 @@ describe('crossOriginRPC', function() {
 
     beforeEach(function() {
       fakeStore = {
-        searchUris: sinon.stub().returns('THE_SEARCH_URIS'),
```

Comment:
Can you help me understand why this gets removed in this diff?

Reply:
I'm not 100% on this, but I think was originally one of the old registered methods done the old way. This is from cross-origin-rpc. All I did was make this a function called `registerMethods` and pulled it out outside of the server function so it's a bit more clear where this API is defined in that file, but is what the older code looked like

```
    // The set of methods that clients can call.
    let methods = {
      searchUris: store.searchUris,
    };

    let method = methods[request.method];
```

So the tests, naturally tested this stuff, but we were never using `searchUris` from the rpc server anymore, so this basically got replaced with `changeFocusModeUser`

Might good to confirm that with Rob, but that was my take away from the call last week